subsys/net/l2/openthread/openthread.c

```diff
@@ -96,6 +96,18 @@ static struct net_linkaddr *ll_addr;
 
 static struct net_mgmt_event_callback ip6_addr_cb;
 
+#include <power/reboot.h>
+#include <stdio.h>
+int devicedata_get_pskd(const char **);
```

Comment:
These additions look very "tight", we could put empty line after the includes, and before the devicedata_get_pskd()